### PR TITLE
Fixing monsterType register script for scripted spells

### PR DESCRIPTION
**Pull request by debone**
- Fixed Marid as sample monster with missing scripted spell

Another probable problem with the converted XML monsters.

### CODE REVIEW

**Comment by heberholanda** [comment]
Great job.

File: data/scripts/monsters/magicals/marid.lua [hunk 1]
```diff
@@ -96,8 +96,8 @@ monster.attacks = {
 	{name ="speed", interval = 2000, chance = 15, speedChange = -650, duration = 1500},
 	{name ="drunk", interval = 2000, chance = 10, range = 7, shootEffect = CONST_ANI_ENERGY, target = false},
 	{name ="outfit", interval = 2000, chance = 1},
-	{name ="combat", interval = 2000, chance = 15, range = 5, target = false},
 	{name ="combat", interval = 2000, chance = 15, minDamage = -30, maxDamage = -90, type = COMBAT_ENERGYDAMAGE, effect = CONST_ME_ENERGYHIT, target = false}
```
Comment (MajestyOTBR) [inline data/scripts/monsters/magicals/marid.lua:99]: Missing comma in line 99.

Comment (majestyotbr) [inline data/scripts/monsters/magicals/marid.lua]: Missing comma in the end of line 99.

File: data/scripts/monsters/magicals/marid.lua [hunk 2]
```diff
@@ -96,8 +96,8 @@ monster.attacks = {
...
+	{script ="djinn electrify", interval = 2000, chance = 15, range = 5},
```
Comment (majestyotbr) [inline data/scripts/monsters/magicals/marid.lua]: Remove comma in the end of line 100.

**Review by majestyotbr — APPROVED** [review]

**Pull request merged**